README.md

```diff
@@ -97,8 +97,21 @@ access different parts of the GitLab API. For example, to list all
 users:
 
 ```go
-git := gitlab.NewClient(nil, "yourtokengoeshere")
-//git.SetBaseURL("https://git.mydomain.com/api/v4")
+git, err := gitlab.NewClient(nil, "yourtokengoeshere")
```

Comment:
Here and in a few other places there are still examples of passing `nil` http client to this function.

Reply:
Thanks, updated...